File: 7_kyu/Is_n_divisible_by.lua
local solution = {}

-- Checks if n is divisble by all following arguments.
function solution.isDivisible(n, ...)
  local args = {...}
  for _,d in ipairs(args) do
    if n % d ~= 0 then
      return false
    end
  end
  return true
end

return solution

File: 7_kyu/Is_n_divisible_by.py
def is_divisible(*args) -> bool:
    """
    Returns true if first number is divisble by the other arguments.
    """
    if not args or len(args) < 1:
        # none or only one argument
        return True
    return all( args[0] % d == 0 for d in args[1:] )
